package.json

```diff
@@ -2,41 +2,26 @@
   "name": "yeoman-generator-list",
   "description": "Server for the Yeoman generator list on yeoman.io/generators",
   "repository": "yeoman/yeoman-generator-list",
-  "keywords": [
-    "generator",
-    "github",
-    "keyword",
-    "list",
-    "npm",
-    "package",
-    "packages",
-    "server",
-    "yeoman"
-  ],
   "author": "Yeoman",
   "license": "BSD",
   "dependencies": {
-    "compression": "^1.4.3",
-    "connect": "^3.4.0",
-    "connect-timeout": "^1.4.0",
-    "errorhandler": "^1.3.0",
-    "gcloud": "^0.23.0",
-    "gh-got": "^2.0.0",
-    "github-url-to-object": "^1.5.2",
-    "got": "^4.2.0",
-    "mkdirp": "^0.5.1",
-    "morgan": "^1.5.0",
-    "newrelic": "^1.22.0",
-    "npm-keyword": "^3.0.0",
-    "package-json": "^1.2.0",
-    "q": "^1.4.1"
-  },
-  "devDependencies": {
-    "mocha": "^2.2.5"
+    "gcloud": "^0.25.1",
+    "gh-got": "^2.3.0",
+    "github-url-to-object": "^2.1.0",
+    "newrelic": "^1.24.0",
+    "npm-keyword": "^4.2.0",
+    "package-json": "^2.2.1"
   },
+  "devDependencies": {},
   "engines": {
     "node": ">=4.0.0"
   },
-  "start": "node server.js",
+  "main": "generator.js",
+  "files": [
```

Comment:
i thought it automatically was since its in main. This could probably be removed since private is true